sdk/confidentialledger/Azure.Security.ConfidentialLedger/tests/samples/HelloWorldSamples.cs

```diff
@@ -167,57 +148,58 @@ public void HelloWorld()
             while (status == "Pending")
             {
                 statusResponse = ledgerClient.GetTransactionStatus(transactionId);
-                status = JsonDocument.Parse(statusResponse.Content)
-                    .RootElement
-                    .GetProperty("state")
-                    .GetString();
+                status = statusResponse.Content.ToDynamic().state;
             }
 
             // The ledger entry written at the transactionId in firstResponse is retrieved from the default collection.
             Response getResponse = ledgerClient.GetLedgerEntry(transactionId);
 
             // Try until the entry is available.
             loaded = false;
-            JsonElement element = default;
             contents = null;
             while (!loaded)
             {
-                loaded = JsonDocument.Parse(getResponse.Content)
-                    .RootElement
-                    .TryGetProperty("entry", out element);
+                dynamic json = getResponse.Content.ToDynamic();
+
+                // TODO: Not currently handled
+                //loaded = (LedgerEntry)json.entry != null;
```

Comment:
Previously, we returned a "null" JsonData if a property was missing, but this is not yet implemented in this new version.

Reply:
What is the current behavior?

Reply:
Right now, if you access a property that isn't present in the JSON, it will throw, as shown in the few lines below this commented out line (which I added to implement the functionality using the current behavior).  Our goal is to enable checks for optional properties that look like strongly-typed models, so we want to be able to check for an absent property without an exception being thrown.  Further details here: https://github.com/Azure/azure-sdk-for-net/pull/33063#discussion_r1089532143